tests/unit/test_dataset_geo.py

```diff
@@ -537,8 +537,9 @@ def update_callback(request, snoop):
                 "record": {"geom": {"point": [1, 1]}, "id1": "2", "id2": "b"},
             },
         ]
-        expected = "\n".join(json.dumps(update) for update in updates)
-        actual = snoop["payload"]
+        expected = [json.dumps(update).replace('"', "'") for update in updates]
+        self.maxDiff = None
```

Comment:
@mollysacks what does setting `maxDiff` here do?

Reply:
that was so I could look at the entire list, I forgot to delete it but I can go back and do that

Reply:
yes, please do delete 😄